### PR TITLE
Create Xnest

### PR DESCRIPTION
**Pull request by hhirsch**

### CODE REVIEW

File: pages/linux/Xnest [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Xnest
+
+> Start a nested x-server
+
+- Start nested x-server as :1 and start a terminal inside
+
+`Xnest :1 -ac &`
+
+`xterm -display :1`
```
Comment (igorshubovych) [inline pages/linux/Xnest]: Why is this command?

**Comment by hhirsch** [comment]
@igorshubovych Why is this command what?

File: pages/linux/Xnest (hunk 1 above)
Comment (igorshubovych) [inline pages/linux/Xnest]: You have `Xnest` command example, and then `xterm` example without explanation.

**Comment by G570972680893** [comment]
😎💜

**Comment by hhirsch** [comment]
@igorshubovych You can use the terminal to start any number of other processes inside Xnest.

**Comment by felixonmars** [comment]
I'm still not convinced that `xterm -display :1` belongs here, since it's an example of using an application-specific option to choose an alternative display, which is not limited to Xnest either.

**Comment by hhirsch** [comment]
@felixonmars Yeah. Whatevs.

**Pull request closed without merging**